README.md

```diff
@@ -25,6 +25,7 @@ The first words of an issue's title will typically indicate the project it invol
 - [Juju](https://juju.is/docs):  open source orchestration engine
 - [Canonical Kubernetes](https://ubuntu.com/kubernetes/docs): the reference platform for Kubernetes on all major public clouds
 - [LXD](https://documentation.ubuntu.com/lxd/en/latest/): open source container and VM management at any scale
+- [Anbox Cloud](https://anbox-cloud.io/docs): Solution offering scalable Android in the cloud
```

Comment:
I feel this list is "kinda" alphabetical - should this one go first (and maybe fix the few lines that are in the wrong order)?